src/test/java/seedu/address/model/person/NricMatchesPredicateTest.java

```diff
@@ -0,0 +1,63 @@
+package seedu.address.model.person;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import org.junit.jupiter.api.Test;
+
+import seedu.address.testutil.PersonBuilder;
+
+public class NricMatchesPredicateTest {
+
+    @Test
+    public void equals() {
+        Nric nric = new Nric("T1234567A");
+        Nric nric2 = new Nric("T1234567B");
```

Comment:
I don't think we should be using numbers in our variable names? Could consider naming it something like `firstNric` and `secondNric`